README.md

```
@@ -28,102 +42,144 @@ https://minimemory.jp/
-| コンテナ | Docker (compose.yml + Dockerfile.dev) |
-| CI/CD | GitHub Actions（Brakeman / RuboCop / RSpec / SimpleCov 90%+） |
-| 環境変数 | dotenv-rails |
+| テスト | RSpec |
+| 開発環境 | Docker |
+| CI/CD | GitHub Actions（Brakeman / RuboCop / RSpec / SimpleCov ） |
 | ホスティング | Render |
 
-## ■画面遷移図
+## ✅技術選定の理由
+
+  #### 言語 / FW: Ruby / Ruby on Rails
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)